packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -438,15 +439,22 @@ preSubmitInfo: {
   field: 'privacyAgreementAccepted',
   label: <span>I have read and accept the <a href="/privacy">privacy policy</a>.</span>,
   error: 'You must accept the privacy policy before continuing',
+  // optional override
+  CustomComponent: CustomPreSubmitComponent
 }
 ```
-
+#### Default `PrivacyAgreement`
 ![Required checkbox before form submission](../../images/forms/Required-Checkbox.jpg)
 
-### Usage guidelines
+#### `CustomComponent` example
+![Required checkbox before form submission](../../images/forms/preSubmitInfo-customComponent.jpeg)
+
+#### Usage guidelines
```

Comment:
```suggestion
### Usage guidelines
```
Looks like we actually skipped a heading level for all these. :man_facepalming: The one above this is
```md
## Required checkbox before form submission
```